app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
```suggestion
    def __init__(
                 self,
                 name: str,
                 health: int = 100,
                 hidden: bool = False
                 ) -> None:
```

Reply:
ok

Reply:
When I doing it. I have a mistake.

Reply:
I took it all back.

Reply:
And i have not a mistake.

Reply:
everything is OK on my machine
not just do like I mentioned but reformat code a bit
![image](https://github.com/mate-academy/py-herbivores-and-carnivores/assets/86322722/a79a38e2-5522-4923-b074-d5465385172a)

Reply:
Sorry, I am stupid. All is good.